deployments/bridges/rialto-millau/dashboard/grafana/provisioning/dashboards/relay-millau-to-rialto-messages-dashboard.json

```
@@ -804,6 +804,12 @@
       "interval": "",
       "legendFormat": "Unconfirmed rewards at Rialto",
       "refId": "A"
+      },
+      {
+        "expr": "(scalar(max_over_time(Millau_to_Rialto_MessageLane_00000000_lane_state_nonces{type=\"source_latest_confirmed\"}[2m])) - scalar(max_over_time(Millau_to_Rialto_MessageLane_00000000_lane_state_nonces{type=\"target_latest_confirmed\"}[2m]))) * (max_over_time(Millau_to_Rialto_MessageLane_00000000_lane_state_nonces{type=\"target_latest_received\"}[2m]) > bool min_over_time(Millau_to_Rialto_MessageLane_00000000_lane_state_nonces{type=\"target_latest_received\"}[2m]))",
+        "interval": "",
+        "legendFormat": "Unconfirmed rewards at Rialto (zero if no messages are not being delivered)",
```

I find this a little confusing. What do you think of this?

```suggestion
        "legendFormat": "Unconfirmed rewards at Rialto (zero if messages are being delivered to Millau)",
```

---

"no ... not" is definitely confusing :) But I'm also confused about your suggestion - what do you mean by "zero if messages are being delivered to Millau" in context of Millau -> Rialto lane? Probably there's confusion with PR description? Basically - the graph (and alert) stays the same. But if there are no new delivery transactions, then the graph used in alert drops to zero => so alert is not activated.

I think the correct suggestion would probably be be "zero if messages are not being delivered" or "zero if messages are not being delivered to Rialto"?

---

```suggestion
        "legendFormat": "Unconfirmed rewards at Rialto (zero if messages are not being delivered to Millau)",
```

Let me suggest a middle ground for you :)